inference_rules.adoc

```diff
@@ -349,7 +351,10 @@ For each of the following benchmarks it is necessary to use the following infere
 |Summarization (GPT-J) |min_new_tokens |30 | Minimun number of new tokens to generate
 |Summarization (GPT-J) |max_new_tokens |128 | Maximum number of new tokens to generate
 |Summarization (GPT-J) |early_stopping |True | Use the EOS token to stop generating tokens
+|Summarization (Llama2) |min_new_tokens |2 | Minimun number of new tokens to generate
```

Comment:
I think llama2 min_new_tokens is 1 https://github.com/mlcommons/inference/blob/c71b83f92f0cd2c3b2f07ffec9d3108bb227b712/language/llama2-70b/SUT_API.py#L37
Mixtral is 2